### ISSUE TITLE
[ADD] VBA services (batch 1) to Service Taxonomy

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#15748]
### Which Taxonomy is this term for?

VA services taxonomy

### Proposed term name/label

Label | Machine Name | Health Service API ID
-- | -- | --
Benefit letters and decision letters | benefit-letters-and-decision-letters | BenefitLetters
Burials and memorials | burials-and-memorials | BurialClaimAssistance
Disability compensation | disability-compensation | DisabilityClaimAssistance
Education benefits for family members | education-benefits-for-family | FamilyEducationBenefits
Education benefits for Veterans | education-benefits-for-veterans | EducationClaimAssistance
Home loans | home-loans | VAHomeLoanAssistance
Life Insurance | life-insurance | InsuranceClaimAssistanceAndFinancialCounseling
Military records | military-records | MilitaryRecords
Veterans Pension | veterans-pension | Pensions

### Does this overlap with any existing term in the taxonomy?

No

### Rationale for adding this term

These terms are supported by multiple rounds of research as part of the [VBA Regional Office MVP](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10514).

### Which products will this term impact?

VBA Regional Office

### Have stakeholders from impacted products been consulted?

Yes, Michelle has communicated with VBA and also gave VAMC a heads up

### Have you drafted content for all the relevant subfields?

- Original source of truth for all VBA services [VBA Services](https://dvagov.sharepoint.com/:x:/s/VBAVA.govmodernization/ET964xEqUFREirhtu34HQVIBbsNOu12QIUc-EVNF_aW98g) (requires VA network)
- This content has been copy edited by CAIA 
- And reviewed for accuracy with VBA stakeholders

### Acceptance Criteria (Sprint 98) - 5 points

- [x] Review/resolve feedback from ROs 
- [x] Draft machine names & API IDs
- [x] Review machine names & API IDs with @swirtSJW & @omahane 
- [x] Consult with @mmiddaugh & @aklausmeier about need for Taxonomy Council involvement / approval
- [x] If approved, a Drupal admin from Product team may add the term to the taxonomy
- [x] Content for subfields is added in Drupal by Product team
  - [x] [Benefit letters and decision letters](https://prod.cms.va.gov/health-care/benefit-letters-and-decision-letters) 
  - [x] [Burials and memorials](https://prod.cms.va.gov/health-care/burials-and-memorials)
  - [x] [Disability compensation](https://prod.cms.va.gov/health-care/disability-compensation) 
  - [x] [Education benefits for family members](https://prod.cms.va.gov/health-care/education-benefits-for-family-members) 
  - [x] [Education benefits for Veterans](https://prod.cms.va.gov/health-care/education-benefits-for-veterans)
  - [x] [Home loans](https://prod.cms.va.gov/health-care/home-loans) 
  - [x] [Life Insurance](https://prod.cms.va.gov/health-care/life-insurance)
  - [x] [Military records](https://prod.cms.va.gov/health-care/military-records)
  - [x] [Veterans Pension](https://prod.cms.va.gov/health-care/veterans-pension) 

~~### Follow up approval and launch steps (could be a separate ticket)~~ moved to #16445

**Comment by davidmpickett** [comment]
Exporting current state of excel document 
[VBA Services.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13480386/VBA.Services.xlsx)

**Comment by davidmpickett** [comment]
I have consolidated the excel doc down to only the 11 services under consideration for batch 1 and put the RO feedback in main tab
[VBA_Services_batch1.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13506113/VBA_Services_batch1.xlsx)

A nice validation that we made good choices in selecting the first batch is that all 11 services were marked as "This is a core service offering at my location" by RO respondent(s)

**Comment by davidmpickett** [comment]
> Member

@swirtSJW  Corollary question for lighthouse. Will anything break if some of these services ID don't have a service on our end?

For instance we won't be using `ApplyingForBenefits` for anything, because it's actually a task that applies to all benefit services, not a service in itself.

**Comment by davidmpickett** [comment]
Pulling these two terms out of batch 1 because they need more discussion on the actual term name
- Burial in a VA cemetery
- Compensation benefits for family members

**Comment by davidmpickett** [comment]
I updated the source of truth doc on sharepoint. I renamed the column "roadmap" to "GitHub ticket tracking this term." Adding static copy of that doc for quick reference:
[VBA_Services_2023_12_1.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13531171/VBA_Services_2023_12_1.xlsx)

**Comment by davidmpickett** [comment]
Also, gathered up all the terms that didn't already have a ticket into a new one: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16269

**Comment by davidmpickett** [comment]
Added draft content for all VBA subfields for Veterans Pension - https://prod.cms.va.gov/health-care/veterans-pension

**Comment by xiongjaneg** [comment]
@davidmpickett In the meeting, you mentioned we might need a spike ticket for discovery post-MVP about care vs. service "at such-and-such Regional Office." Would you stub that please? We can discuss at our 1:1 if needed too. Thank you!

**Comment by davidmpickett** [comment]
@xiongjaneg Already stubbed - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15690

**Comment by jilladams** [comment]
From Adam: 
> Each list of services from the Swagger comes from a different `enum` in the codebase. I believe when we get a service payload, we do a lookup of the service id to see which enum it is valid in then add the service to the appropriate service array. So it should just know when it gets an `ApplyingForBenefits` id that it belongs in the `benefits` list.

**Comment by jilladams** [comment]
@swirtSJW could you comment here today to clarify if an admin could go ahead and make the changes Dave listed, based on Adam's response? And Dave / Swirt, if you need help from an Admin, I'm available.

**Comment by swirtSJW** [comment]
I think @omahane has said he would grab this.

**Comment by swirtSJW** [comment]
I just added these.  However I got to wondering about the casing on the api ids.   Have these been approved by LH?

**Comment by davidmpickett** [comment]
@jilladams @xiongjaneg - this is going to continue into sprint 99 due to blockers and my thin capacity in 98

**Comment by jilladams** [comment]
@xiongjaneg need a follow up ticket here.

**Comment by xiongjaneg** [comment]
3 pts completedin 98, 2 pts in 99

**Comment by davidmpickett** [comment]
Work in progress - notes on implementation

Label | Notes
-- | --
[Benefit letters and decision letters](https://prod.cms.va.gov/health-care/benefit-letters-and-decision-letters) | 
[Burials and memorials](https://prod.cms.va.gov/health-care/burials-and-memorials) |  I left the Regional service header & description fields empty as this is not a service that has regional centers
[Disability compensation](https://prod.cms.va.gov/health-care/disability-compensation) | 
[Education benefits for family members](https://prod.cms.va.gov/health-care/education-benefits-for-family-members) | 
[Education benefits for Veterans](https://prod.cms.va.gov/health-care/education-benefits-for-veterans) | 
[Home loans](https://prod.cms.va.gov/health-care/home-loans) |
[Life Insurance](https://prod.cms.va.gov/health-care/life-insurance) | 
[Military records](https://prod.cms.va.gov/health-care/military-records) | 
[Veterans Pension](https://prod.cms.va.gov/health-care/veterans-pension) |

**Comment by davidmpickett** [comment]
Made good progress, but didn't quite get this across the line today

**Comment by mmiddaugh** [comment]
@davidmpickett thank you for the helpful visual - it will support understanding with a variety of audiences

For efficiency, I suggest we seek VBA input before requesting CAIA input - hopefully that will avoid additional back and forth

Question: Are these accurate expectations for VBA service fields? 
- A given service has one facility header and description
- A given service `may` have one regional service header and description 
- A given service has one online self-service link (but no more)

**Comment by davidmpickett** [comment]
> @davidmpickett thank you for the helpful visual - it will support understanding with a variety of audiences
> 
> For efficiency, I suggest we seek VBA input before requesting CAIA input - hopefully that will avoid additional back and forth - is this set ready for VBA input?
> 
> Question: Are these accurate expectations for VBA service fields?
> 
> * A given service has one facility header and description
> * A given service `may` have one regional service header and description
> * A given service has one online self-service link (but no more)

- Yes, that is how these fields are currently configured AND
- None of the fields are required, so technically any of these fields could be empty. 
  - For example, there may be cases where a Facility Header & VBA Description are sufficiently explanatory and a Facility Description is redundant / not needed. (See "Military records" and "Benefit letters and decision letters" for two cases where we could make that argument.)
- If there is a strong case for supporting multiple self-service links, we could consider increasing the cardinality of this field in the future. I think for MVP we should focus on the single best link, rather than opening the door to lots of links.

**Comment by davidmpickett** [comment]
> For efficiency, I suggest we seek VBA input before requesting CAIA input - hopefully that will avoid additional back and forth - is this set ready for VBA input?

@mmiddaugh @xiongjaneg  I think we should resolve [this defect](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16378) before sending the links for VBA. If y'all approve that as a sprint injection I can have a PR ready to merge today.

**Comment by davidmpickett** [comment]
@mmiddaugh @aklausmeier Now that bug fix has merged, these are ready to send to VBA for review / approval. Let me know if you need me to help in drafting any communication around that.

I have listed out the **Follow up approval and launch steps** after the ACs. It is up to y'all and @xiongjaneg @jilladams whether you want to use this ticket to track that, or separate that out into a VBA Launch checklist or something else.

**Comment by xiongjaneg** [comment]
@jilladams Let's talk at our 1:1 when we talked about talking about launch. Thank you!

**Comment by jilladams** [comment]
Ported those notes to the new ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16445

**Comment by xiongjaneg** [comment]
@davidmpickett verified and closed defect ticket #16378 
Moved pre-launch list to #16445

Is there anything else in this ticket awaiting PO/Prod review?

**Comment by mmiddaugh** [comment]
Ready to close from my perspective, thank you!

**Comment by davidmpickett** [comment]
@mmiddaugh of the [9 terms](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15748#issuecomment-1850961454) that you took to VBA for review, the only one [I've seen feedback on](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16269#issuecomment-1879295820) is Disability Compensation (which I [implemented](https://prod.cms.va.gov/taxonomy/term/1144/revisions/view/2147/2179/visual_inline) today)

Does that mean the other 8 are all good from VBA's perspective and ready to send to CAIA for final approval? Or are you still waiting on feedback?

cc @aklausmeier
